core-services/image-mirroring/openshift/mapping_origin_4_11

```diff
@@ -37,6 +37,7 @@ registry.ci.openshift.org/origin/4.11:baremetal-operator quay.io/openshift/origi
 registry.ci.openshift.org/origin/4.11:baremetal-runtimecfg quay.io/openshift/origin-baremetal-runtimecfg:4.11 quay.io/openshift/origin-baremetal-runtimecfg:4.11.0 quay.io/openshift/origin-baremetal-runtimecfg:latest
 registry.ci.openshift.org/origin/4.11:base quay.io/openshift/origin-base:4.11 quay.io/openshift/origin-base:4.11.0 quay.io/openshift/origin-base:latest
 registry.ci.openshift.org/origin/4.11:build-test-qemu-img quay.io/openshift/origin-build-test-qemu-img:4.11 quay.io/openshift/origin-build-test-qemu-img:4.11.0 quay.io/openshift/origin-build-test-qemu-img:latest
+registry.ci.openshift.org/origin/4.11:capk-oc-bin-image quay.io/openshift/origin-capk-oc-bin-image:4.11 quay.io/openshift/origin-capk-oc-bin-image:4.11.0 quay.io/openshift/origin-capk-oc-bin-image:latest
```

Comment:
@bparees

Reply:
> What is `capk`? I would assume this is not an image we publish (in which case it shouldn't show up)

@smarterclayton - We do need it for the hypershift project, so it could support kubevirt as a cluster api provider.

Reply:
Removed. Thanks @smarterclayton